include/boost/histogram/detail/probit.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_DETAIL_PROBIT_HPP
+#define BOOST_HISTOGRAM_DETAIL_PROBIT_HPP
+
+#include <cmath>
+
+# define PI 3.14159265358979323846  /* pi */
+
+namespace boost {
+namespace histogram {
+namespace detail {
+
+template <class T>
+T ErfInv(T x){
+   T tt1, tt2, lnx, sgn;
+   sgn = (x < 0) ? -1.0f : 1.0f;
+
+   x = (1 - x)*(1 + x);        // x = 1 - x*x;
+   lnx = logf(x);
+
+   tt1 = 2/(PI*0.15449436008930206298828125) + 0.5f * lnx;
```

Comment:
`PI` is not available on all platforms. The best is to replace PI*0.15449436008930206298828125 by 0.4853583466776075

Reply:
Or better just replace 2/(...) by 4.120666747961526